DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -0,0 +1,80 @@
+﻿CREATE VIEW AdminUsers AS
+SELECT dbo.Users.ID                                      AS AdminID,

```

Comment:
Should this be the AdminAccounts.AdminID?

Reply:
Yep

Reply:
changed